examples/safe-ui/src/platform.rs

```diff
@@ -0,0 +1,293 @@
+// Copyright © SixtyFPS GmbH <info@slint.dev>
+// SPDX-License-Identifier: MIT
+
+extern crate alloc;
+use alloc::boxed::Box;
+use alloc::rc::Rc;
+//use alloc::vec::Vec;
+//use core::cell::RefCell;
```

Comment:
```suggestion
```

But there are a lot of other commented code in this file so perhaps this is on purpose?

Reply:
I wanted to leave the queue code in, hope to find a solution for that still.

I tried to use critical section but I couldn't get it to link. Then I realized that the clean thing would be to abstract that away and use freertos mutex. But that's tricky or ugly to bridge, and the issue of what the right way of implementing invoke_from_event_loop is remains. Perhaps we need a second method (invoke_from_event_loop_isr or so)

All in all I concluded that this is beyond the PoC.